projects/plugins/jetpack/extensions/shared/components/audio-player/style.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:meta';
+@use 'sass:string';
```

Comment:
Looks like a `@use 'sass:color'` got left out here?

Reply:
Yep, still "In Progress" for a bit more. I saw some deprecated notices that I'm going to handle as well.